numpy/tests/test_ctypeslib.py

```diff
@@ -3,9 +3,10 @@
 import sys
 
 import numpy as np
-from numpy.ctypeslib import ndpointer, load_library
+from ctypes import c_int, cast, POINTER
```

Comment:
I don't think this is OK, based on #8898 requiring `ctypes` to not be imported at the module level. Do any other tests import ctypes at this level?